### ISSUE TITLE
positioning bug

### ISSUE THREAD

**Issue by nacee-hu** [snapappointments/bootstrap-select#2482]
If the selectpicker is in popover, and the dropdown menu container is "body", then the positioning fail.

I found a posible solution (or bug in code):
in `selectPosition` function, the `if (version.major < 4 || display === 'static') {...}` code line change to `if (version.major >= 4 || display === 'static') {...}`, the **<** sign change to **>=** sign, or flip condition sides.

operating system: ubuntu,win
browser(s),: chrome,ff
jQuery version: 3.4.1
Bootstrap version: 4.3.1
bootstrap-select version: 1.13.17

Sorry but I can't create a reduced test case with include a link to a live example.
And sorry for my poor english.

**Comment by caseyjhol** [comment]
The code you mentioned is intentional; it's supposed to use Popper.js for positioning if the version is 4 or greater, otherwise it uses bootstrap-select's internal positioning logic.

**Comment by nacee-hu** [comment]
Yes, I was expecting what you wrote too, but unfortunately the condition is exactly the opposite. So not version 4 and higher, but smaller than version 4. Excerpt from the code:

https://github.com/snapappointments/bootstrap-select/blob/a9f8afaa9120016334cf0218a9db5e1803166756/js/bootstrap-select.js#L2197
https://github.com/snapappointments/bootstrap-select/blob/a9f8afaa9120016334cf0218a9db5e1803166756/js/bootstrap-select.js#L2198

It also appears in the comment that it is 4+, but in the condition it is the opposite. Or am I wrong?

**Comment by caseyjhol** [comment]
bootstrap-select's internal positioning logic is only used if the version is less than 4 or if `data-display="static"`. So, the condition is correct.
